vue/src/client/services/ApiService.ts

```diff
@@ -482,6 +483,7 @@ export class ApiService {
   }
 
   public static patchSiteEvaluation(id: string, data: SiteEvaluationUpdateQuery): CancelablePromise<boolean> {
+    // We clear the cache so it will reload the tiles and display the new color.
```

Comment:
Is this comment misplaced?

Reply:
I removed that I think the `state.filters.randomKey` plus the comment above it indicates the clearing of the tile cache.
I may have meant that on the server it clears the vector tile cache when items are patched, but I think that's clear in the endpoint in django now.